### ISSUE TITLE
Nextgen Gallery with Elementor issue

### ISSUE THREAD

**Issue by kv1953** [elementor/elementor#7443]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

I have used Nextgen Gallery with Elementor for over a year but the new release of Elementor causes an error preview could not be loaded. All  add-ons work ok.

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
Thanks for reporting,
Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information: 

* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.


Although, we are familiar with this issue and a patch will be released soon.

Thanks.

**Comment by kv1953** [comment]
[system-info-www.seisc.co.uk-13-03-2019.txt](https://github.com/elementor/elementor/files/2965579/system-info-www.seisc.co.uk-13-03-2019.txt)

**Comment by shilo-ey** [comment]
This issue has been fixed in 2.5.6

Feel free to update!
